<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yildiz's Toolkit</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="defectListLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Defect list</string>
    </property>
   </widget>
   <widget class="QLabel" name="TemperatureLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doscarFileBrowse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="DefectFileFormatLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Defect file format</string>
    </property>
   </widget>
   <widget class="QLabel" name="DefectFileFormatExample">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>e.g.   defects-*.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DOSCAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="dopingConcenLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doping Concentration</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Output">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>370</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="defectListInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DefectFileFormatInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="doscarLocation">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dopingConcentrationPpm">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ppm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dopingConcentrationInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QSlider" name="temperatureSlider">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>18</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="temperatureInput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="oxygenPressure">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Oxygen partial pressure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minOxygenP">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxOxygenP">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="oxygenPressureAtm">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>atm</string>
    </property>
   </widget>
   <widget class="QLabel" name="oxygenPressure_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="oxygenPressure_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QWidget" name="mplwindow" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>531</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="Output_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>DefectFileFormatInput</tabstop>
  <tabstop>defectListInput</tabstop>
  <tabstop>temperatureSlider</tabstop>
  <tabstop>temperatureInput</tabstop>
  <tabstop>doscarLocation</tabstop>
  <tabstop>doscarFileBrowse</tabstop>
  <tabstop>dopingConcentrationInput</tabstop>
  <tabstop>minOxygenP</tabstop>
  <tabstop>maxOxygenP</tabstop>
  <tabstop>calculateButton</tabstop>
  <tabstop>calculateButton_2</tabstop>
  <tabstop>Output</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
